typescript-style-frontend.js

```diff
@@ -4,6 +4,7 @@ module.exports = {
             files: ['*.ts', '*.tsx'],
             rules: {
                 // @typescript-eslint/eslint-plugin Supported rules
+                "no-empty-function": "off",
```

Comment:
Why you off `no-empty-function` instead `@typescript-eslint/no-empty-function`?

Reply:
https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/no-empty-function.md
![image](https://user-images.githubusercontent.com/60327711/161209499-fdae9021-6a45-41e6-bdfd-70771d25e6d0.png)
i tried @typescript-eslint/no-empty-function: "off" - it doesn't work